quickstarts/azure/azure-time-series-insight/config.yml

```diff
@@ -0,0 +1,40 @@
+slug: azure-time-series-insight
```

Comment:
Should the slug be `azure-time-series-insights` to match the name of the service?